aws/sdk/examples/medialive-helloworld/src/main.rs

```diff
@@ -0,0 +1,16 @@
+#[tokio::main]
+async fn main() -> Result<(), medialive::Error> {
+    let client = medialive::Client::from_env();
+    let input_list = client.list_inputs().send().await?;
+
+    if let Some(inputs) = input_list.inputs {
+        inputs.iter().for_each(|i| {
+            let input_arn = i.arn.to_owned().unwrap_or_default();
+            let input_name = i.name.to_owned().unwrap_or_default();
+
+            println!("Input Name : {}, Input ARN : {}", input_name, input_arn);
+        })
+    }
```

Comment:
```suggestion
    for input in input_list.inputs.unwrap_or_default() {
          let input_arn = input.arn.as_deref().unwrap_or_default();
          let input_name = input.name.as_deref().unwrap_or_default();
          println!("Input Name : {}, Input ARN : {}", input_name, input_arn);
    }
```

Reply:
same pattern can be used in the other example

Reply:
`as_deref()` goes from `Option<T>` to `Option<&T>` when allows you do unwrap without taking ownership

Reply:
Awesome, let me make the changes.